languages/rust/exercises/concept/numbers/.meta/example.rs

```diff
@@ -0,0 +1,15 @@
+const PRODUCTION_RATE_DEFAULT: u32 = 221;
+
+pub fn production_rate_per_hour(speed: u8) -> f64 {
+    match speed as u32 {
+        0 => 0.0,
+        s @ 1..=4 => (PRODUCTION_RATE_DEFAULT * s) as f64,
```

Comment:
I'm still learning Rust so I could be *way* off here. Do students need to know about Ranges and the `@` operator within `match` expressions to solve it in this manner?

Reply:
Quite possibly. Maybe it's a good idea to add another example that uses if's. This does throw a wrench in the wheels when it comes to a minimal amount of sensible solutions. We may have to check for conditionals and different implementations of pattern matching.

Reply:
Good point. We can get back to that as I don't want to block progress.